ios-drag-drop.js

```diff
@@ -316,7 +318,8 @@
   }
 
   function once(el, event, handler, context) {
-    if(context) handler = handler.bind(context)
+    if(context)
+      handler = handler.bind(context);
```

Comment:
Which would you prefer? Single line w/ semicolon or brackets? I thought it was a mistake at first with it being an if statement on a single line with no semicolon.